### DISCUSSION TITLE
Welcome to mosaik Discussions!

### DISCUSSION THREAD

**Discussion by Tobii42** [OFFIS-mosaik/mosaik#1] [Announcements]
## 👋 Welcome!

We are proud to announce that you can now find us on GitHub 🥳. We are especially excited to welcome you to the Discussions feature (you have already found it!) where we want to get in contact with you 💬. You have a question, a feature request or want to generally discuss the amazing possibilities with co-simulation and mosaik? We are curious to hear from you! We still have our mailing list, but we see GitHub as the better place to exchange problems, solutions, and ideas. See you there!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eike** [comment]
Moin (as we would say here in the north of Germany)!

I’m Eike and I’m currently one of the core developers of mosaik. I’m hoping to help users of mosaik and to use their feedback to improve it. So ask away!
